modules/getting-started/pages/build-a-hello-world-application.adoc

```diff
@@ -56,23 +57,24 @@ The HTTP connector allows your Mule application to connect to Web resources thro
 
 . In the canvas, click the HTTP connector to open its Properties Editor.
 . Click the plus sign image:plus-1.png[plus] to add a Connector Configuration with a host and port number.
-. In the Global Element Properties dialog, set Host to *localhost*, accept the default port *8081*, click OK.
+. In the Global Element Properties dialog, set Host to *localhost*, accept the default port *8081*, click *OK*.
 +
 image::http-global-config.png[]
 +
-The properties window opens, with Path populated with " / " and Connector Configuration populated with a reference to the global element you just created.
+The properties window opens, with Path populated with " / " with *Connector Configuration* populated with a reference to the global element you just created.
```

Comment:
3 with in one sentence hard to read. The properties window opens. Path is populated with .... and Connector Configuration is populated with... There's probably a better way to do it.